Header header
int64 number
geometry_msgs/PoseArray body_poses
# Unidenfied markers. (E.g., the single marker used for calibration).	
Mocap/marker_set uid_markers
# The set of identified markers.
Mocap/marker_set[] id_marker_sets